examples/constant_liar.py

```diff
@@ -0,0 +1,31 @@
+"""
+Example for parallel optimization with skopt.
+The points to evaluate in parallel are selected according to the "constant lie" approach.
+
+"""
+
+import numpy as np
+from multiprocessing.pool import ThreadPool
+from skopt.space import Real
+from skopt.learning import GaussianProcessRegressor
+from skopt import Optimizer
+
+# ThreadPool is used for parallel computations
+pool = ThreadPool()
```

Comment:
I drop the ThreadPool, and use joblib instead, I think this way it is also consistent with the rest of the code in repository. I think it is  good to have example with actual parallel computation going on, as then all user needs to do is substitute the function of their own to have things run in parallel.